tests/mock_web_api_server.py

```diff
@@ -121,18 +122,19 @@ def set_common_headers(self):
     def _handle(self):
         parsed_path: ParseResult = urlparse(self.path)
         path = parsed_path.path
+        self.server.queue.put(path)
         self.received_requests[path] = self.received_requests.get(path, 0) + 1
         try:
             if path == "/webhook":
                 self.send_response(200)
-                self.set_common_headers()
+                self.set_common_headers(len("OK"))
                 self.wfile.write("OK".encode("utf-8"))
                 return
 
-            if path == "/received_requests.json":
+            if path == "/health":
```

Comment:
Are you using this endpoint? If we no longer need this, you can simply remove it.

Reply:
Good point 💯  I did add this to try and see if it would be more reliable to ensure the `mock_web_api_server` responds with a success response before running the tests, there was no notable difference here